irohad/ordering/impl/ordering_gate_cache/on_demand_cache.cpp

```diff
@@ -87,3 +87,7 @@ void OnDemandCache::rotate() {
   auto second_element_it = boost::next(circ_buffer.begin());
   circ_buffer.rotate(second_element_it);
 }
+
+size_t OnDemandCache::availableTxsCapacity() const {
+  return max_cache_size_ - circ_buffer.back().first;
```

Comment:
.first ? is it correct? 

should it be something like?
```
max_cache_size_ - boost::size(circ_buffer.back())
```

Reply:
please take a look at the definition:
https://github.com/hyperledger/iroha/blob/54cb1aceef1f95d0c465a85c14187d787b46d049/irohad/ordering/impl/ordering_gate_cache/on_demand_cache.hpp#L45-L51
So `circ_buffer` element is a pair of total txs number and batches set. What we need here is the transactions number, so we take the first pair element.